lib/vmpooler/pool_manager.rb

```diff
@@ -462,7 +460,7 @@ def _check_snapshot_queue
 
     def migration_limit(migration_limit)
       # Returns migration_limit setting when enabled
-      return false if migration_limit == 0 || ! migration_limit
+      return false if migration_limit == 0 || !migration_limit # rubocop:disable Style/NumericPredicate
```

Comment:
Disable the cop here because migration limit can come in as an Integer OR boolean type